src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -59,17 +59,19 @@
                                lang="en">
                         <label class="custom-file-label" for="import-csv">{{ importedFilename }}</label>
                     </div>
-                    <div v-if="validationErrors.length"
-                         id="import-validation-errors"
-                         class="text-danger small error-message mt-2 ">
-                        <div class="col-sm-2 d-inline-block"></div>
-                        <div class="col-sm-6 d-inline-block pl-0">
-                            Unable to import from csv:
-                            <div v-for="error in validationErrors" class="import-validation-error ml-2">
+                    <b-alert :show="!!validationErrors.length"
```

Comment:
This works well but not aligned with the rest of the input elements.

![Screenshot 2021-11-29 at 10 53 22](https://user-images.githubusercontent.com/4240748/143857395-a8995be1-55e3-42ea-989c-bb4e2e49f0e1.png)

Reply:
It's displaying at the same position and width as the git change warning, which seemed consistent to me. Would you rather it was only as wide as the input element itself and vertically aligned with it? I think that might look a bit cramped but I can give it a try.

Reply:
Only if it looks better otherwise ignore, I am not too keen, looking great already.